wpilibj/src/athena/java/edu/wpi/first/wpilibj/DigitalGlitchFilter.java

```diff
@@ -52,10 +52,14 @@ public void free() {
   }
 
   private static void setFilter(DigitalSource input, int channelIndex) {
-    if (input != null) { // Counter might have just one input
-      DigitalGlitchFilterJNI.setFilterSelect(input.m_port, channelIndex);
+    if (input != null) { // Counter might have just oe input
+      // analog triggers are not supported for DigitalGlitchFilters
+      if (input.getAnalogTriggerForRouting()) {
+        return;
```

Comment:
Maybe actually. Before it was just passing a digital port down, which was definitely wrong. I wonder if there is a way for AnalogTriggers to work with glitch filters. Would have to ask Joe about that.